pep-9999.rst

```diff
@@ -259,6 +259,73 @@ No::
         forward_reference: 'OtherClass'
     class OtherClass: ...
 
+Version and Platform Checks
+---------------------------
+
+Type stubs for libraries that support multiple Python versions can use version
+checks to supply version-specific type hints. Type stubs for different Python
+versions should still conform to the most recent Python version's syntax, as
+explain in the Syntax_ section above.
+
+Version checks are if-statements that use ``sys.version_info`` to determine the
+current Python version. Version checks should only check against the ``major`` and
+``minor`` parts of ``sys.version_info``.
+
+Yes::
+
+    if sys.version_info >= (3,):
+      # Python 3-specific type hints.
+    else:
+      # Python 2-specific type hints.
+
+    if sys.version_info.major >= 3:
```

Comment:
Does this currently work? I think I prefer that we allow *only* tuple-based comparisons, so that type checkers don't have to support tuple and `.major`-based operations.

Reply:
> Does this currently work?
Depends on the type checker, I suppose, but it's part of PEP 484: https://www.python.org/dev/peps/pep-0484/#version-and-platform-checking

>I think I prefer that we allow _only_ tuple-based comparisons, so that type checkers don't have to support tuple and `.major`-based operations.

I'm fine with this, personally. I think tuple-based comparisons are better overall, and I've opened #26 so we can discuss this.

Reply:
Added recommendation for tuple-based comparisons.